File: EXAMEN_SOAP_DOTNET_BDD/01 SERVIDOR/02_SERVIDOR_COMERCIALIZADORA/Comercializadora_Soap_Dotnet/ec.edu.monster.modelo/RespuestaVenta.cs
using System.Runtime.Serialization;

namespace Comercializadora_Soap_Dotnet.ec.edu.monster.modelo
{
    [DataContract(Name = "respuestaVenta")]
    public class RespuestaVenta
    {
        [DataMember(Order = 1)]
        public bool Exitoso { get; set; }

        [DataMember(Order = 2)]
        public string Mensaje { get; set; }

        [DataMember(Order = 3)]
        public int IdFactura { get; set; }

        [DataMember(Order = 4)]
        public double Total { get; set; }

        [DataMember(Order = 5)]
        public string FormaPago { get; set; }

        [DataMember(Order = 6)]
        public double Descuento { get; set; }

        [DataMember(Order = 7)]
        public int IdCreditoBanco { get; set; }

        [DataMember(Order = 8)]
        public double CuotaMensual { get; set; }

        [DataMember(Order = 9)]
        public int NumeroCuotas { get; set; }

        public RespuestaVenta()
        {
            Exitoso = false;
            Mensaje = "";
            IdFactura = 0;
            Total = 0;
            FormaPago = "";
            Descuento = 0;
            IdCreditoBanco = 0;
            CuotaMensual = 0;
            NumeroCuotas = 0;
        }
    }
}

File: PROYECTO_RESTFULL_JAVA_BDD/02 CLIESC/ComercializadoraEscritorio/src/main/java/ec/edu/monster/models/RespuestaVenta.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.monster.models;

/**
 *
 * @author josue
 */
public class RespuestaVenta {
    private boolean exito;
    private int idFactura;
    private int idCreditoBanco;
    private double cuotaMensual;
    private int numeroCuotas;
    private String mensaje;
    
    public RespuestaVenta() {}
    
    // Getters y Setters
    public boolean isExito() { return exito; }
    public void setExito(boolean exito) { this.exito = exito; }
    
    public int getIdFactura() { return idFactura; }
    public void setIdFactura(int idFactura) { this.idFactura = idFactura; }
    
    public int getIdCreditoBanco() { return idCreditoBanco; }
    public void setIdCreditoBanco(int idCreditoBanco) { this.idCreditoBanco = idCreditoBanco; }
    
    public double getCuotaMensual() { return cuotaMensual; }
    public void setCuotaMensual(double cuotaMensual) { this.cuotaMensual = cuotaMensual; }
    
    public int getNumeroCuotas() { return numeroCuotas; }
    public void setNumeroCuotas(int numeroCuotas) { this.numeroCuotas = numeroCuotas; }
    
    public String getMensaje() { return mensaje; }
    public void setMensaje(String mensaje) { this.mensaje = mensaje; }
}
